index.html

```diff
@@ -1933,6 +1924,19 @@ <h3>Verification Material</h3>
 an example of a public key with a compound key identifier.
             </p>
           </dd>
+          <dt><dfn>publicKeyMultibase</dfn></dt>
+          <dd>
+            <p>
+The <code>publicKeyMultibase</code> property is OPTIONAL. This feature is
+non-normative. If present, the value MUST be a <a
+data-cite="INFRA#string">string</a> representation of a [[?MULTIBASE]] encoded
+public key.
+            </p>
+            <p class="advisement">
+There might be some cases where <code><b>public</b>KeyMultibase</code> can
```

Comment:
```suggestion
Note that publicKeyMultibase is not a standard and is therefore subject to change. There might be some cases where <code><b>public</b>KeyMultibase</code> can
```

Reply:
@selfissued per your request.

I will leave this suggestion here so folks can thumbs up or thumbs down.